custom_components/plex_assistant/localize.py

```diff
@@ -955,57 +953,97 @@
             "achte": "8",
             "neunte": "9",
             "zehnte": "10",
-            # Do not edit the keys of pre and post
-            "pre": ["die",],
+
+            "pre": [
+                "die",
+            ],
             "post": [],
         },
-        # Keywords, Pre, and Post
+
         "season": {
-            "keywords": ["staffel"],
+            "keywords": [
+                "staffel",
+            ],
             "pre": [
                 "ste",
                 "te",
-                "die"],
-            "post": ["von"],
+                "die",
+            ],
+            "post": [
+                "von",
+            ],
         },
         "episode": {
-            "keywords": ["episode", "folge"],
+            "keywords": [
+                "episode",
+                "folge",
+            ],
             "pre": [
                 "ste",
                 "te",
-                "die"],
-            "post": ["von"],
+                "die",
+            ],
+            "post": [
+                "von",
+            ],
         },
         "latest": {
             "keywords": [
-                  "neuste", 
-                  "aktuellste", 
-                  "letzte", 
-                  "aktuelle"
-              ],
-            "pre": ["die"]
```

Comment:
A comma was removed here after `["die"]`. This is enough to cause errors. Found a few things like this. They are much easier to see if you stick to the formatting of the template.